PyInstaller/utils/hooks/qt/__init__.py

```diff
@@ -588,6 +588,11 @@ def collect_qtnetwork_files(self):
         def _ssl_enabled(package):
             import importlib
 
+            # Create QCoreApplication instance (to suppress warnings)
+            # equivalent to: from package import QtCore
+            QtCore = importlib.import_module('.QtCore', package)
+            QtCore.QCoreApplication()
```

Comment:
We should probably keep the reference to the constructed `QCoreApplication` object until the function (which is isolated program, really) finishes.

Reply:
Agreed. Is there any naming convention for such unusued variable to avoid linter warnings? I'd probably choose `_application` otherwise.

Reply:
I'd name it `app`, because that's why I usually name it in my PySide programs. And if the linter gets fussy about it (and it probably will, because it is an unused local variable...), just stick a `  # noqa: F841` at the end of that line.

Reply:
Also, I think `PyQt` bindings require `args` list (`sys.argv`) to be passed to `QCoreApplication` constructor. We can probably get away with passing an empty list here, but we do need to explicitly pass the argument in order to keep code compatible with both `PyQt` and `PySide`.